sql/triggers.sql

```diff
@@ -18,26 +19,27 @@ BEGIN
   END IF;
 
   IF TG_OP = 'INSERT' THEN
-    changes := row_to_json(NEW);
+    state := row_to_json(NEW);
   ELSIF TG_OP = 'UPDATE' THEN
-    changes := row_to_json(NEW);
+    state := row_to_json(NEW);
```

Comment:
is this correct? Does `NEW` contain all state or only the new fields/values?

Reply:
NEW and OLD variables contain all state of new record and old record accordingly. (ref: https://www.ibm.com/docs/en/ias?topic=plsql-trigger-variables)